shaders/texture_mipmap_generation/slang/texture.vert.slang

```diff
@@ -0,0 +1,46 @@
+/* Copyright (c) 2025, Sascha Willems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 the "License";
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+struct VSInput
+{
+    float3 Pos : POSITION0;
+    float2 UV;
+};
+
+struct UBO
+{
+    float4x4 projection;
+    float4x4 model;
```

Comment:
I think, `modelview`, instead of `model`, like it's named in the glsl shader, would be a better name.